.travis.yml

```diff
@@ -133,4 +133,4 @@ before_install:
     - cd ../
 script:
     # Build KLEE and run tests
-    - ${KLEE_SRC}/.travis/klee.sh
+    - travis_wait 20 ${KLEE_SRC}/.travis/klee.sh
```

Comment:
Don't do this. Show output of the build process instead. If something goes wrong you want to know it fast.

From https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

> We recommend careful use of travis_wait, as overusing it can extend your build time when there could be a deeper underlying issue. When in doubt, file a ticket or email us first to see if something could be improved about this particular command first.

Reply:
@MartinNowack Good point, although the issue is related to the system tests, not the build process. No output can be shown when a solver test takes too long. Nevertheless, I replaced `travis_wait` with another workaround that will not interfere with other things.